scripts/fast-reboot

```diff
@@ -156,6 +156,42 @@ function request_pre_shutdown()
     }
 }
 
+function recover_issu_bank_file_instruction()
+{
+    error "To recover (${ISSU_BANK_FILE}) file, do the following:"
+    error "$ docker exec -it syncd sx_api_dbg_generate_dump.py"
+    error "$ docker exec -it syncd cat /tmp/sdkdump | grep 'ISSU Bank'"
+    error "Command above will print the VALUE of ISSU BANK - 0 or 1, use this VALUE in the next command"
+    error "$ printf VALUE > /host/warmboot/issu_bank.txt"
+}
+
+function check_issu_bank_file()
+{
+    ISSU_BANK_FILE=/host/warmboot/issu_bank.txt
+    MLNX_ISSU_BANK_BROKEN=102
+
+    if [[ ! -s "$ISSU_BANK_FILE" ]]; then
+        error "(${ISSU_BANK_FILE}) does NOT exist or empty ..."
+        recover_issu_bank_file_instruction
+        exit "${MLNX_ISSU_BANK_BROKEN}"
+    fi
+
+    issu_file_chars_count=`wc -m ${ISSU_BANK_FILE} | awk '{print $1}'`;
```

Comment:
>wc -m ${ISSU_BANK_FILE} | awk '{print $1}' [](start = 27, length = 42)

A more simple solution
```
stat -c %s FILE
```

Reply:
Did you try? This comment is still applicable.

---
In reply to: [427512540](https://github.com/Azure/sonic-utilities/pull/915#discussion_r427512540) [](ancestors = 427512540)

Reply:
> Did you try? This comment is still applicable.
> 
> In reply to: [427512540](https://github.com/Azure/sonic-utilities/pull/915#discussion_r427512540) [](ancestors = 427512540)

Fixed